pkg/app/pipedv1/plugin/azure/deployment/func_swap.go

```diff
@@ -0,0 +1,71 @@
+package deployment
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/pipe-cd/pipecd/pkg/app/pipedv1/plugin/azure/config"
+	"github.com/pipe-cd/pipecd/pkg/app/pipedv1/plugin/azure/provider"
+	"github.com/pipe-cd/pipecd/pkg/plugin/sdk"
+)
+
+func (p *Plugin) executeAzureFuncSwapStage(ctx context.Context, dts []*sdk.DeployTarget[config.AzureDeployTargetConfig], input *sdk.ExecuteStageInput[config.AzureApplicationSpec]) sdk.StageStatus {
+	lp := input.Client.LogPersister()
+	lp.Info("Start azure func swap")
+	if len(dts) != 1 {
+		lp.Errorf("Currently support only one deployment target, instead got %d", len(dts))
+		return sdk.StageStatusFailure
+	}
+	sdkClient, err := provider.NewAzureClient(ctx, dts[0].Config, lp)
+	if err != nil {
+		lp.Errorf("Failed to create Azure Functions sdkClient: %v", err)
+		return sdk.StageStatusFailure
+	}
+	sdkClient.SetResourceTags(map[string]*string{
+		provider.LabelManagedBy:   to.Ptr(provider.ManagedByPiped),
+		provider.LabelPiped:       to.Ptr(input.Request.Deployment.PipedID),
+		provider.LabelCommitHash:  to.Ptr(input.Request.TargetDeploymentSource.CommitHash),
+		provider.LabelApplication: to.Ptr(input.Request.Deployment.ApplicationID),
+	})
```

Comment:
It's better to make a blank line between logical blocks for readability.
```suggestion
	})
	
```